tools/cloudharness-test/cloudharness_test/apitest_init.py

```diff
@@ -1,44 +1,62 @@
 import os
 import logging
+import requests
 
 import schemathesis as st
 
 from cloudharness.auth import get_token
 
-if "APP_URL" in os.environ:
-
-    app_url = os.environ["APP_URL"]
-
-    try:
-        openapi_uri = app_url + "/openapi.json"
-        logging.info("Using openapi spec at %s", openapi_uri)
-        schema = st.from_uri(openapi_uri)
-    except Exception as e:
-        raise Exception(f"Cannot setup api tests: {openapi_uri} not reachable. Check your deployment is up and configuration") from e
+if "APP_URL" or "APP_SCHEMA_FILE" in os.environ:
+    app_schema = os.environ.get("APP_SCHEMA_FILE", None)
+    app_url = os.environ.get("APP_URL", "http://samples.ch.local/api")
+    logging.info("Start schemathesis tests on %s", app_url)
+    if app_schema:
+        openapi_uri = app_schema
+        schema = st.from_file(openapi_uri)
```

Comment:
@filippomc for Django REST I would not rely on the openapi.yaml file in the git repo to be the latest. Django REST auto generates the openapi specs on runtime